tests/test_tree.py

```diff
@@ -597,3 +597,14 @@ def normalize(x):
             lazy = tree[branchname].lazyarray(keycache=keycache, basketcache=basketcache)
             for i in range(len(lazy), 0, -1):
                 self.assertEqual(normalize(lazy[i - 1 : i + 3]), strict[i - 1 : i + 3].tolist())
+
+    def test_hist_in_tree(self):
+        tree = uproot.open("tests/samples/Event.root")["T"]
+        check = [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0,
```

Comment:
Looks a little messy. Not sure how to improve this portion.

Reply:
Oh! I was just a little concerned that it was a bit too crude. You can merge it in if the code is okay :)